app/renderer/lib/client-frameworks/ruby.js

```diff
@@ -55,6 +55,10 @@ driver.quit`;
   codeFor_back () {
     return `driver.back`;
   }
+
+  codeFor_clickElement () {
```

Comment:
yeah this is ok for now, but probably we don't want to need to handle `clickElement` when we already handle `click`